### PR TITLE
Feature/#58 add and remove movies to from custom lists frontend

### PR DESCRIPTION
**Pull request by revodark**

### CODE REVIEW

File: frontend/src/components/AddToList/AddToList.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Dialog } from '@material-ui/core';
```
Comment (Rolly007) [inline frontend/src/components/AddToList/AddToList.jsx]: Maybe if is the unique import from material ui library you can import directly => `import Dialog from '@material-ui/core/Dialog';`

File: frontend/src/components/CardsPagination/CardsPagination.jsx [hunk 2]
```diff
@@ -21,12 +21,14 @@ const DIALOG_ACCEPT_LABEL = 'Delete';
...
-  list,
   search,
+  listType,
+  listId,
   sortBy = 'popularity.desc',
   requestPath,
   moviesPagination,
-  listsPagination
+  listsPagination,
+  withDelete
```
Comment (Rolly007) [inline frontend/src/components/CardsPagination/CardsPagination.jsx:31]: I am not sure but I tink that the word with + "something" is for implement hight order components, please could you change the name of this param

File: frontend/src/components/CardsPagination/CardsPagination.test.js [hunk 3]
```diff
@@ -137,6 +137,7 @@ describe('testing the CardsPagination on listPagination mode', () => {
           list='custom_list'
           requestPath='/users/movies/lists'
           listsPagination
+          withDelete
```
Comment (Rolly007) [inline frontend/src/components/CardsPagination/CardsPagination.test.js:140]: same of the above comment

File: frontend/src/components/PaginationCard/PaginationCard.jsx [hunk 4]
```diff
@@ -51,7 +51,8 @@ function PaginationCard({
   totalItems,
   listCard,
   movieCard,
-  onDelete
+  onDelete,
+  withDelete
```
Comment (Rolly007) [inline frontend/src/components/PaginationCard/PaginationCard.jsx:55]: same of the above observation

File: frontend/src/components/AddToList/AddToList.jsx (hunk 1 above)
Comment (revodark) [reply frontend/src/components/AddToList/AddToList.jsx]: thanks!

File: frontend/src/components/CardsPagination/CardsPagination.jsx (hunk 2 above)
Comment (revodark) [reply frontend/src/components/CardsPagination/CardsPagination.jsx:31]: You're totally right! thanks!

File: frontend/src/components/CardsPagination/CardsPagination.test.js (hunk 3 above)
Comment (revodark) [reply frontend/src/components/CardsPagination/CardsPagination.test.js:140]: thanks!

**Pull request merged**